PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -122,7 +136,7 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
         case .item(let objectID):
             select(item: objectID)
         case .filterButton(let filterID):
-            apply(filter: filterID, from: cellID)
+            apply(filter: filterID, from: cellID, listOptions: listOptions)
```

Comment:
Do we need to pass `listOptions` here still? It should already be accessible as a property